debian/control

```diff
@@ -0,0 +1,20 @@
+Source: libkdumpfile
+Section: misc
+Priority: optional
+Maintainer: Delphix Engineering <eng@delphix.com>
+Standards-Version: 4.1.2
+Build-Depends: autoconf,
+               automake,
+               git,
```

Comment:
I'm guessing you just copied that list from linux-pkg, but do you know what is git used for? I'm asking because I want to make sure it doesn't try to fetch so external code dependencies during the build.

Reply:
I don't think it is actually used. Removing it and retesting.